includes/modules/pages/password_forgotten/header_php.php

```diff
@@ -66,14 +66,26 @@
 
     // handle 3rd-party integrations
     $zco_notifier->notify('NOTIFY_PASSWORD_FORGOTTEN_CHANGED', $email_address, $check_customer->fields['customers_id'], $new_password);
+      if (PADSS_ADMIN_SESSION_TIMEOUT_ENFORCED) {
+          $messageStack->add_session('login', SUCCESS_PASSWORD_SENT, 'success');
+      } else {
+          $messageStack->add_session('login', SUCCESS_PASSWORD_SENT_CONFIRM, 'success');
+      }
+      zen_redirect(zen_href_link(FILENAME_LOGIN, '', 'SSL'));
 
   } else {
     $zco_notifier->notify('NOTIFY_PASSWORD_FORGOTTEN_NOT_FOUND', $email_address);
+      if (PADSS_ADMIN_SESSION_TIMEOUT_ENFORCED) {
+          $messageStack->add_session('login', SUCCESS_PASSWORD_SENT, 'success');
+          zen_redirect(zen_href_link(FILENAME_LOGIN, '', 'SSL'));
+      } else {
+          $messageStack->add_session('password_forgotten', EMAIL_ADDRESS_NOT_FOUND);

```

Comment:
drB, i assume you are talking about line 82?  i have no problem changing that, but it does work as expected and i did test it.  i did change the class to `password_forgotten`, as in this situation where the user enters an invalid email address, we could like to stay on this page, which it currently does. 

i did now just try it with `add` as opposed to `add_session`.  they both work fine.  i have no problem changing it to `add`.